### PR TITLE
Scrollbar: scrollbar styled and normalized

### PR DESCRIPTION
**Pull request by LFBarreto**
Scrollbars visuals aren't consistent between platforms.
We can style them using css webkit apis or default to basic scrollbar if css doesn't support it

localhost_8080_webpack_index html

Scrollbars appears on hover on scroll zones and dont take space thanks to scroll mode overlay (supported on webkit only and fallback to scroll otherwise"

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow

### Type

UI Polish

### Parts of the app affected / Test plan

Modals with scrollable content
Scroll zones
DropDown with scroll

**needs testing on mac, windows and linux**

### CODE REVIEW

File: src/renderer/styles/theme.js [hunk 1]
```diff
@@ -130,6 +130,31 @@ const animations = {
...
+  x: css`
+    overflow-y: scroll;
+    overflow-x: overlay;
+    &:hover {
+      --track-color: ${p => p.theme.colors.palette.text.shade30};
+    }
+  `,
+  y: css`
+    overflow-y: scroll;
+    overflow-y: overlay;
```
Comment (gre) [inline src/renderer/styles/theme.js:143]: why is it defining twice?

File: src/renderer/styles/theme.js [hunk 2]
```diff
@@ -130,6 +130,31 @@ const animations = {
...
+    &:hover {
+      --track-color: ${p => p.theme.colors.palette.text.shade30};
+    }
+  `,
+  xy: css`
+    overflow: scroll;
+    overflow: overlay;
```
Comment (gre) [inline src/renderer/styles/theme.js:151]: same

**Review by gre — COMMENTED** [review]
nice

File: src/renderer/styles/theme.js (hunk 1 above)
Comment (LFBarreto) [reply src/renderer/styles/theme.js:143]: its a css fallback on scroll if overlay can't be applied :+1: 
overlay is a deprecated API only working on webkit and maybe it won't work all the time so fallback is necessary in this case to ensure scroll always works.

**Review by MortalKastor — APPROVED** [review]
👏

File: src/renderer/styles/theme.js [hunk 3]
```diff
@@ -130,6 +129,31 @@ const animations = {
   fadeInGrowX: props => css`${fadeInGrowX} ${animationLength} ${easings.outQuadratic} forwards`,
 };
 
+const overflow = {
+  x: css`
+    overflow-y: scroll;
```
Comment (LFBarreto) [inline src/renderer/styles/theme.js]: overflow-x 🚫

**Pull request merged**